src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java

```diff
@@ -127,7 +127,7 @@ public void MercuryNavigationSideTest_001_Open_Navigate_Close() {
         true
     );
 
-    nav.clickOverlay();
+    nav.tapOverlay();
```

Comment:
can this be changed to waitAndClick?
we know chrome mobile have problems with touch events
and from user perspective click here does the same thing